overlord/devicestate/systems.go

```diff
@@ -429,7 +429,8 @@ func createSystemForModelFromValidatedSnaps(model *asserts.Model, label string,
 			bootWith.GadgetSnapOrDir = sn.Path
 		}
 	}
-	if err := boot.MakeRecoverySystemBootable(boot.InitramfsUbuntuSeedDir, recoverySystemDirInRootDir, bootWith); err != nil {
+	// TODO: set optional command line
```

Comment:
what more do we need to here? this will use the gadget defaults from the snaps? did you want to use the current values as well?

Reply:
It was a leftover I think, I've removed now